templates/partials/common_headers.html.jinja2

```diff
@@ -0,0 +1,92 @@
+<meta name="HandheldFriendly" content="True">
+<meta name="MobileOptimized" content="320">
+<meta name="viewport" content="width=device-width, initial-scale=1"> <!-- allows zoom -->
+
+
+<!-- Mobile IE allows us to activate ClearType technology for smoothing fonts for easy reading -->
+<meta http-equiv="cleartype" content="on">
+
+
+<link rel="shortcut icon" href="../../favicon.ico" />
+<link rel="apple-touch-icon" href="../../icon.png"/>
+<link rel="icon" href="../../icon.png"/>
+
+<link rel="stylesheet" type="text/css" href="../../css/styles.css" />
+
+
+<!-- For Facebook -->
+<meta property="og:image" content="https://beachboyslegacy.com/{{item["parent"]["image"]}}"/>
+<meta property="og:url" content="http://beachboyslegacy.com/items/{{item["parent"]["uniqueId"]}}"/>
+<meta property="og:site_name" content="Beach Boys Legacy"/>
+<meta property="og:type" content="website"/>
+<meta property="fb:app_id" content="514938358714366">
+
+<!-- for Twitter -->
+<meta name="twitter:image" content="https://beachboyslegacy.com/{{item["parent"]["image"]}}" />
+<meta name="twitter:card" content="summary_large_image" />
+<meta name="twitter:site" content="@beachboyslegacy">
+<meta name="twitter:image:alt" content="Beach Boys Legacy">
+
+
+<!-- For Google+ -->
+<!-- <html itemscope itemtype="http://schema.org/Article"> -->
+
+<meta itemprop="image" content="https://beachboyslegacy.com/{{item["parent"]["image"]}}">
+
+
+<!--[if lt IE 9]>
+  <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+<![endif]-->
+
+
+<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<!-- <script src="../../js/jquery.3.2.1.js"></script> -->
+
+
+
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-10359362-22', 'auto');
+  ga('send', 'pageview');
+  setTimeout("ga('send','event','User explores the site','time on page more than one minute')",60000);
+</script>
+
+<script src="https://embed.widgetpack.com/widget.js"></script>
+<script type="text/javascript">
+  function wpac_ajax_init() {
+      WPac.init({widget: 'Rating', id: 2265, el: 'wpac-rating-ajax'});
+  }
+</script>
+
+
+
+<link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css" />
+<script src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js"></script>
+<script>
+window.addEventListener("load", function(){
+window.cookieconsent.initialise({
+  "palette": {
+    "popup": {
+      "background": "#000"
+    },
+    "button": {
+      "background": "#f1d600"
+    }
+  },
+  "content": {
+    "href": "cookies-policy.html"
+  }
+})});
+</script>
```

Comment:
We are already calling the cookies plugin from post_scripts so I think it can be removed from here

Reply:
I'll move it to post scripts.